lib/kube/proxy/forwarder.go

```diff
@@ -297,9 +292,10 @@ type Forwarder struct {
 	close context.CancelFunc
 	// ctx is a global context signaling exit
 	ctx context.Context
-	// creds contain kubernetes credentials for multiple clusters.
+	// clusterDetails contain kubernetes credentials for multiple clusters.
 	// map key is cluster name.
-	creds map[string]*kubeCreds
+	clusterDetails map[string]*kubeDetails
+	rwMutexDetails sync.RWMutex
```

Comment:
The RWMutex type is used for this mutex but in the code there is no any reference to  RLock/RUnlock calls